locust/web.py

```diff
@@ -285,7 +285,7 @@ def swarm() -> Response:
                     if isinstance(parsed_options_value, bool):
                         parsed_options_dict[key] = value == "true"
                     elif parsed_options_value is None:
```

Comment:
Maybe we can do:
```
elif parsed_options_value is None and value is None:
  parsed_options_dict[key] = None
```
This way we still get the type casting from the else statement